src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -71,23 +71,23 @@ public PersonCard(Person person, int displayedIndex) {
             name.setGraphic(genderIcon);
             name.setContentDisplay(ContentDisplay.RIGHT);
         }
-        phone.setText(person.getPhone().value);
+        phone.setText("Phone: " + person.getPhone().value);
         if (person.getPhone().value.isEmpty()) {
             phone.setManaged(false);
         }
-        nationality.setText(person.getNationality().value);
+        nationality.setText("Nationality: " + person.getNationality().value);
         if (person.getNationality().value.isEmpty()) {
             nationality.setManaged(false);
         }
-        email.setText(person.getEmail().value);
+        email.setText("Email: " + person.getEmail().value);
         if (person.getEmail().value.isEmpty()) {
             email.setManaged(false);
         }
-        tutorialGroup.setText(person.getTutorialGroup().value);
+        tutorialGroup.setText("Tutorial Group: " + person.getTutorialGroup().value);
         if (person.getTutorialGroup().value.isEmpty()) {
             tutorialGroup.setManaged(false);
         }
-        remark.setText(person.getRemark().value);
+        remark.setText("Remarks: " + person.getRemark().value);
```

Comment:
Just to check: For now, there isn't a case where "Remarks: "/"Phone: "/etc.. shows up right? Maybe after delete or edit?

Reply:
I just tested with the modify functions and it looks okay! It behaves the same way as before, if the field is not included, the field label also doesn't appear.